src/main/java/com/myMoneyTracker/app/MyMoneyTrackerApplication.java

```diff
@@ -9,4 +13,14 @@
     public static void main(String[] args) {
         SpringApplication.run(MyMoneyTrackerApplication.class, args);
     }
+    
+    @Bean
```

Comment:
We've decided to use xml files for declaring beans, not annotations. Please create a new spring config file for the application and put there the bean declaration